class TrackingError:
    def __init__(self):
        ...

